doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -397,7 +397,9 @@ Thread samples
 Zigbee samples
 --------------
 
-|no_changes_yet_note|
+* Added:
+
+  * The possibility to configure transmission power for the light switch sample.
```

Comment:
```suggestion
* :ref:`zigbee_light_switch_sample` sample:

  * Added the option to configure transmission power.
```

Reply:
"for the light switch sample"?

Reply:
Ah, I think you missed the other line. It should be:
```
* :ref:`zigbee_light_switch_sample` sample:

  * Added the option to configure transmission power.
```